pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.sonatype.nexus.plugins</groupId>
         <artifactId>nexus-plugins</artifactId>
-        <version>2.8.0-05</version>
+        <version>2.12.1-01</version>
```

Comment:
If we upgrade this, we should only upgrade to 2.11.0, since the plugin should be compatible with 2.11.0 and up. The other possibility is to release a new version 1.2 for 2.12, which supports newer features like the new hashes.